lib/passwordless/token_generator.rb

```diff
@@ -0,0 +1,23 @@
+require 'openssl'
+
+module Passwordless
+  class TokenGenerator
```

Comment:
I think we should split out the hashing from this. The token generators are supposed to be easy to provide yourself and if you do, I think we should still provide the hashing.

Let's let the token generator still just return a simple, unhashed string.